wrphttp/handler.go

```diff
@@ -81,6 +90,12 @@ func WithBefore(funcs ...MessageFunc) Option {
 	}
 }
 
+func WithAfter(funcs ...MessageFunc) Option {
```

Comment:
Definitely need to make sure this new functional option is covered in unit tests.